plugins/layer-count.js

```diff
@@ -64,43 +64,73 @@ plugin.layerCount.pnpoly = function(latlngs, point) {
 	return c;
 }
 
+// Code for calculating the area of a triangle on the surface of a sphere.
+plugin.layerCount.fieldarea = function(LatLngs) {
+	if (LatLngs.length !== 3 || window.plugin.drawTools === undefined) {
+		return 0.0;
+	}
+  return L.GeometryUtil.geodesicArea(LatLngs)
+};
+
+// Take an area in m² and output a not-too long string
+// Mostly to avoid things like "235239856 m²" for large fields
+plugin.layerCount.prettyAreaString = function(area) {
+  if (area === 0.0) return "";
+  
+	// 1million because we're using parseInt. If we try to go to
+	// km² before this we'll just display "0 km²"
+	if (area < 1000000.0) {
+		return "(" + parseInt(area).toLocaleString() + " m²)";
+	} else {
+		let a = parseInt(area * 100) / 100;
+		return "(" + (a / 1000.0 / 1000.0).toLocaleString() + " km²)";
+	}
+};
+
 plugin.layerCount.calculate = function(ev) {
 	var point = ev.latlng;
 	var fields = window.fields;
 	var layersRes = layersEnl = layersDrawn = 0;
+	let areaFields = areaDrawn = 0.0;
```

Comment:
```suggestion
	var areaFields = 0.0;
	var areaDrawn = 0.0;
```
recommended: split initialition